app/plot_app/configured_plots.py

```diff
@@ -220,9 +234,15 @@ def generate_plots(ulog, px4_ulog, db_data, vehicle_data, link_to_3d_page,
                              y_axis_label='[deg/s]', title=axis_name+' Angular Rate',
                              plot_height='small', changed_params=changed_params,
                              x_range=x_range)
-        data_plot.add_graph([lambda data: (axis+'speed',
-                                           np.rad2deg(data[rate_field_names[index]]))],
-                            colors3[0:1], [axis_name+' Rate Estimated'], mark_nan=True)
+        if is_vtol_tailsitter:
+            try:
```

Comment:
Why is this needed?

Reply:
This is the correction for the rate plots.

Reply:
I was referring only to the `try...except`. I would rather explicitly check for `if rates[axis] is not None:`